web/src/beta/lib/core/engines/Cesium/hooks.ts

```diff
@@ -718,6 +723,36 @@ export default ({
         : 1; // default as 1
   }, [property?.render?.antialias]);
 
+  // explicit rendering
+  const explicitRender = useCallback(() => {
+    const viewer = cesium.current?.cesiumElement;
+    if (!requestingRender?.current || !viewer || viewer.isDestroyed()) return;
+    viewer.scene.requestRender();
+    requestingRender.current = false;
+  }, [requestingRender]);
+
+  useEffect(() => {
+    const viewer = cesium.current?.cesiumElement;
+    if (!viewer || viewer.isDestroyed()) return;
+    viewer.scene.postUpdate.addEventListener(explicitRender);
```

Comment:
We can use [this hooks](https://github.com/reearth/reearth/blob/0b8919e8abeb529f13d3066c24a54cbf7436ff9c/web/src/beta/lib/core/engines/Cesium/hooks/useSceneEvent.ts#L37) here.

Reply:
Updated. Thank you!

Reply:
Sorry i revert this part as we discussed the scene event hook can't be used here.